docroot/modules/custom/mass_bigquery/src/BigqueryStorage.php

```diff
@@ -127,7 +127,7 @@ public function updateRecords(array $ids): bool {
     $time = \Drupal::time()->getRequestTime();
```

Comment:
@tom-fleming  looks good to me, hoverer, I noticed that `$time` is not used in this method. 
probably a good time to remove it?

Reply:
Thanks @dstorozhuk. This specific `$time` variable is still used, but I removed the unused one on line 167

Reply:
oh, ok. I checked the file in PHPStorm, so yes, you are right, it is in another method.